src/CHIPBindings.cc

```diff
@@ -2323,38 +2466,70 @@ hipError_t hipHostMalloc(void **Ptr, size_t Size, unsigned int Flags) {
   assert(PageLockSuccess == 0 && "Failed to page lock memory");
 
   *Ptr = RetVal;
-  RETURN(hipSuccess);
+  return hipSuccess;
+}
+
+hipError_t hipHostMalloc(void **Ptr, size_t Size, unsigned int Flags) {
+  CHIP_TRY
+  CHIPInitialize();
+  RETURN(hipHostMalloc_internal(Ptr, Size, Flags));
   CHIP_CATCH
 }
 
 DEPRECATED("use hipHostMalloc instead")
-hipError_t hipHostAlloc(void **Ptr, size_t Size, unsigned int Flags) {
-  RETURN(hipHostMalloc(Ptr, Size, Flags));
+hipError_t hipMallocHost(void **Ptr, size_t Size) {
+  CHIP_TRY
+  CHIPInitialize();
+  RETURN(hipHostMalloc_internal(Ptr, Size, hipHostMallocDefault));
+  CHIP_CATCH
 }
 
-hipError_t hipFree(void *Ptr) {
+DEPRECATED("use hipHostMalloc instead")
```

Comment:
```suggestion
```
Could be removed. Users won't and see this and developers shouldn't.

Reply:
Agreed, and addressed in https://github.com/CHIP-SPV/chip-spv/pull/486/commits/4adf00e8cf3e4069b556b302226cfc4e9a8a6431